README.md

```diff
@@ -43,23 +43,23 @@ npm install
 npm run build:prod:watch
 ```
 
-You can also manually adjusting your API endpoint settings in `webpack.config.js` by altering the `targets` within `proxy`. For example:
+You can also manually adjusting your API endpoint settings by adding `config/development.js` overriding any of the values in `config/base.json`. For example:
```

Comment:
We should mention that they can work directly against the production instance by setting the `ENV` environment variable to `production`. Since that will probably be the most common pattern for external contributors.